calico-cloud/image-assurance/scan-image-registries.mdx

```diff
@@ -1,317 +1,3 @@
 ---
 description: Identify vulnerabilities in workloads to Kubernetes clusters.
 ---
```

Comment:
Note: This file will be deleted with this. I kept this around to get confirmation before I do that.